kuma/wiki/views/list.py

```diff
@@ -90,6 +95,7 @@ def needs_review(request, tag=None):
     return render(request, 'wiki/list/needs_review.html', context)
 
 
+@block_user_agents
```

Comment:
Should we add https://developer.mozilla.org/en-US/docs/localization-tag and https://developer.mozilla.org/en-US/docs/localization-tag/* to robots.txt?